src/main/java/io/jenkins/plugins/step/WithGCPStep.java

```diff
@@ -162,5 +164,15 @@ public void expand(@NonNull final EnvVars env) {
                 env.overrideAll(envVars);
             }
         }
+
+        private static class PublicCredentialValues {
```

Comment:
ah, it is still on J11 from the parent so cannot use `record`, will need to update it to 5.0+ version (not in scope of this PR ofc)